src/app/modules/pages/aggregate-search/aggregate-search.component.html

```diff
@@ -0,0 +1,31 @@
+<section
+  class="bg-purple-2"
+  fxLayoutAlign="center"
+  fxLayout="row"
+  *ngIf="!aggregateResults.aggregates?.length">
+  <div class="container-no-results" fxLayout="column" fxLayoutGap="18px">
+    <h2>Sua pesquisa não encontrou nenhum conteúdo correspondente</h2>
+    <p>
+      Confira a busca realizada. Verifique a grafia e tente novamente.
+    </p>
+  </div>
+</section>
+
+<div *ngIf="aggregateResults.aggregates?.length">
+  <section class="bg-purple-3" fxLayoutAlign="center" fxLayout="row">
+    <div class="container-results" fxLayout="column" fxLayoutGap="42px">
+      <ng-container>
+        <p>
+          Abaixo, estão os resultados encontrados nos diários do município
+          informado. Os dados estão organizados de maneira compactada e
+          disponível para download!
```

Comment:
Esse texto aparece tanto na página de resultados de UF quanto de município.
```suggestion
          Abaixo, estão os resultados encontrados na localidade informada
          informado. Os dados estão organizados de maneira compactada e
          disponível para download!
```